### ISSUE TITLE
Alpha channel images

### ISSUE THREAD

**Issue by Itamii** [darktable-org/darktable#9301]
I tried using the latest dark table version, but it seems like it is not able to work with alpha channel media whatsoever. 
Whenever I import transparent background images, the background gets filled by darktable in the preview as well as the export.

I looked for settings to solve this as well as tried different source files and formatting.
I could not figure out if this was just not a feature or a bug occurring.

Either way, as we work a lot over here with transparent background images, it would be extremely helpful to be able to work with alpha channel images.

**Comment by parafin** [comment]
This is highly unlikely feature to be implemented IMHO.

**Comment by johnny-bit** [comment]
Transparency has it's own set of problems (see eg. pre-multipled alpha ) that is different can of worms than whole can of worms dealing with colour.

NOT handling transparency in files in darktable is neither feature nor bug. It's just geared towards non-transparent media.

Why would darktable be suitable to handling your images if it had some form of transparency handling? Adding to that - would it be possible for you to eg save alpha channel to separate file then apply alpha to file generated with darktable?

**Comment by johnny-bit** [comment]
@Itamii - Currently we don't understand why you'd need the functionality in darktable and how you'd plan to use it. Please comment with explainer about whys and hows.

**Comment by aurelienpierre** [comment]
Transparency is also used for the infra-red dust detection of film scans. I'm sure there is a way to milk TIFF metadata to know whether the alpha is premultiplied or not.

**Comment by johnny-bit** [comment]
Understood:unclear, no info from originator of issue, no workable info...

Closing.

@Itamii - please add more info on what exactly are you dealing with and how having transparency handling (and what kind of transparency handling) would help you. Then I'll reopen.

**Comment by acebone** [comment]
Transparency would be very useful in conjunction with AI-services that removes backgrounds.

if one could import a PNG with the background replaced by (various degrees of) transparency through the compositemodule and from that create a darktable mask, similar to those created by the parametric mask system, that would be a great booster in productivity.

I often export my photo as tiff into gimp, to be able to seamlessly darken the background 1 EV or so.

I am aware that I can do it in Darktable using eg. the path mask, but it's a lot of work.

Being able to import a png of the photo with the background removed would be a huge timesaver, and offer yet a creative workflow to darktable.

**Comment by AbsurdlySuspicious** [comment]
I've recently had a case where I needed to develop a linear colorspace OpenEXR file with complex alpha exported from Blender: compress it to sRGB and export to 16bit PNG. Sigmoid did a very good job, but with alpha lost I couldn't compose this image later. Eventually I had to approximate tone adjustments in blender compositor, export in openEXR bypassing Blender's color space conversion, then compose and export to PNG in Krita. This worked, but I'd probably get more satisfying results (and a much more sane composition "pipeline") developing this render in darktable
